### DISCUSSION TITLE
Start transloco before the angular application

### DISCUSSION THREAD

**Discussion by sawa-ko** [ngneat/transloco#485] [Q&A] [answered]
How can I achieve that? Because currently I have to use the `selectTranslate()` subscription every time I need to get the translation of something in a ngOnInit method, it is very annoying because if I use the `.translate()` method it tells me that the translation does not exist.

**Comment by shaharkazaz** [comment, marked as answer]
You can init that app after loading the translation by using the [APP_INITIALIZER](https://angular.io/api/core/APP_INITIALIZER) injection token.

**Reply by sawa-ko** [reply]
Is there an example of how to do it with transloco? I can't find one in the documentation

**Reply by shaharkazaz** [reply]
An example already exists. See this [recipe](https://ngneat.github.io/transloco/docs/recipes/prefetch).

**Reply by sawa-ko** [reply]
I have already done it and it is still the same. I did it exactly like the example.
I keep getting the error of translation missing

**Reply by shaharkazaz** [reply]
Share a reproduction :)
